modules/apps/frontend-js/frontend-js-react-web/src/main/resources/META-INF/resources/js/components/page-accessibility-tool/index.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import React, {useState} from 'react';
+
+import Occurrence from './components/Occurrence';
+import Violation from './components/Violation';
+import Violations from './components/Violations';
+import {FilteredViolationsContextProvider} from './components/useFilteredViolations';
+
+function isBetweenRange({childrenCount, newIndex}) {
+	return 0 <= newIndex && newIndex <= childrenCount;
+}
+
+function SidebarPanelsNavigator({children}) {
+	const [activePageIndex, setActivePageIndex] = useState(
+		PagesEnum.Violations
```

Comment:
I think it just declares `number` instead of enum because it is only used here and we can decouple that information from the component block since it should not know which panels it renders